x-pack/plugins/observability/public/pages/overview/loading_observability.tsx

```diff
@@ -5,49 +5,33 @@
  * 2.0.
  */
 
-import React, { useContext } from 'react';
-import styled, { ThemeContext } from 'styled-components';
-import { EuiFlexItem } from '@elastic/eui';
-import { EuiPanel } from '@elastic/eui';
-import { EuiFlexGroup } from '@elastic/eui';
-import { EuiLoadingSpinner } from '@elastic/eui';
-import { EuiText } from '@elastic/eui';
+import { EuiFlexGroup, EuiFlexItem, EuiLoadingSpinner, EuiText } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
-import { WithHeaderLayout } from '../../components/app/layout/with_header';
-
-const CentralizedFlexGroup = styled(EuiFlexGroup)`
-  justify-content: center;
-  align-items: center;
-  // place the element in the center of the page
-  min-height: calc(100vh - ${(props) => props.theme.eui.euiHeaderChildSize});
-`;
+import React from 'react';
+import { ObservabilityHeaderMenu } from '../../components/app/header';
+import { usePluginContext } from '../../hooks/use_plugin_context';
 
 export function LoadingObservability() {
-  const theme = useContext(ThemeContext);
+  const { ObservabilityPageTemplate } = usePluginContext();
 
   return (
-    <WithHeaderLayout
-      headerColor={theme.eui.euiColorEmptyShade}
-      bodyColor={theme.eui.euiPageBackgroundColor}
-    >
-      <CentralizedFlexGroup>
+    <ObservabilityPageTemplate template="centeredBody">
```

Comment:
Minor comment, because this works just fine, but the `<KibanaPageTemplate />` accepts a `isEmptyState` prop, if set to true it will look at which content exists (children, header etc) and determine the correct template itself.

Reply:
That's a good point. I wonder, though, if semantically the loading screen qualifies as an empty state.

@cchaos @katefarrar can you provide any guidance on how to best render full-page loading states with the `<KibanaPageTemplate />`? We're currently setting `template="centeredBody"` to display a centered spinner with accompanying loading text.

Reply:
There isn't anything that the Eui/KibanaPageTemplate provides specifically semantically to indicate empty state. It's purely a visual representation for consistency. Do you have a screenshot of your loading state that I could see for better advice?

Reply:
I think I noticed the same when I was testing it out. The original loading state sat within a small panel, but now with the new page component, I think it'd make more sense to remove the panel around the loading message and show the content area background? With this new layout, we're always expecting to have the main content area there.

<img width="1939" alt="Screenshot 2021-05-27 at 09 00 57" src="https://user-images.githubusercontent.com/4104278/119780629-58f42a80-beca-11eb-8cd7-a5e2715259c2.png">

Reply:
I think it looks ok with just `template="centeredContent"`. What do you think?

![image](https://user-images.githubusercontent.com/973741/119807677-acbf3d80-bee3-11eb-9388-86bdc9acf2eb.png)

Reply:
@weltenwort I wonder where the background is coming from around the spinner and text? If it's a panel, it supports `color="transparent"` to avoid it adding that light grey background.

Reply:
There's no extra panel. The shaded background is also visible in the EUI examples:

![image](https://user-images.githubusercontent.com/973741/119808878-d6c52f80-bee4-11eb-9b9d-7fa11b89fc93.png)

Reply:
Yeah, probably shouldn't go down that route. Let's leave as is, but I'll speak to @cchaos on how we want to handle full page loading states in the new layouts. Thanks for taking a look, and for fixing the original style issue 👍

Reply:
I talked it over with @formgeist and I'll try to build something into the page template. Though it will most likely still follow the same pattern as the empty state, so I'd suggest using that for now until we have something specific for loading.